apps/etterlatte-migrering/src/main/kotlin/migrering/MigrerSpesifikkSak.kt

```diff
@@ -39,12 +52,53 @@ internal class MigrerSpesifikkSak(
             return
         }
 
+        val pesyssak = hentSak(sakId).tilVaarModell().also {
+            pesysRepository.lagrePesyssak(pesyssak = it)
+        }
+        packet.eventName = Migreringshendelser.MIGRER_SAK
+        val request = pesyssak.tilMigreringsrequest()
+        packet.hendelseData = request
+        try {
+            verifiserer.verifiserRequest(request)
```

Comment:
Hadde det vært ryddigere om `MigrerSpesifikkSak` ikke trengte å forholde seg til exceptions fra `Verifiserer`. Ser for meg at det kan bli rotete hvis vi begynner å legge på flere sjekker. Ser for meg en type struktur som enten gir OK, eller feil med en type vi kan lagre ned som inkluderer informasjon om hva som gikk galt.

Reply:
God ide! Kan jo sikkert vera litt forskjellige greier som går feil her etter kvart, og det høyrast ut som noko som gir ryddigare kode.